.travis.yml

```diff
@@ -4,7 +4,9 @@ branches:
     - master
     - develop
 
-language: python
```

Comment:
Python + OSX will break Travis CI.
As we use miniconda, it's fine to skip the language.

Maybe the reason is that I have tested this patch with my repo?